publication-commons/src/main/java/no/unit/nva/publication/model/business/publicationstate/PublishedResourceEvent.java

```diff
@@ -21,7 +21,7 @@ public PublicationLogEntry toLogEntry(SortableIdentifier resourceIdentifier, Log
                    .withResourceIdentifier(resourceIdentifier)
                    .withIdentifier(SortableIdentifier.next())
                    .withTopic(LogTopic.PUBLICATION_PUBLISHED)
-                   .withTimestamp(Instant.now())
+                   .withTimestamp(date)
```

Comment:
this is right date. Not when event is proceeded, despite that these two timestamps are almost similar.